modules/nested-accordion/assets/js/frontend/handlers/nested-accordion.js

```diff
@@ -15,8 +18,23 @@ export default class NestedAccordion extends Base {
 				accordionItems: '.e-n-accordion-item',
 				accordionItemTitles: '.e-n-accordion-item-title',
 				accordionContent: '.e-n-accordion-item > .e-con',
+				itemTitle: '.e-n-accordion-item-title',
```

Comment:
I prefer to add these values inside getKeyboardNavigationSettings(). I can help you to study the correct approach.